### PR TITLE
Feat: remove `strictLimit`

### LINKED ISSUE #32: `strictLimit` seems to be unnecessary
What's the use case for this option? As far as I can see it's just a workaround for incorrect rounding that is happening. 

In `paginate.ts` you're using `Math.round` to calculate `totalPages`; this should be `Math.ceil`, you never want to be in a situation where you're not showing the last page of results because you rounded down.

I'm happy to raise a PR to fix this.

### PR DESCRIPTION
**Pull request by jamesmoss**
- Stop using random numbers in tests, it mean that often the `randomIds` was only ~60 items in length. Just uses incrementing numbers now
- Now wipes the entire database table both before and after all the tests run to ensure a clean slate and old test data isnt left over.
- Removes `strictLimit` - this was just a hacky workaround because total pages wasnt being calculated correctly.
- Fixes a bug where `hasNextPage` was incorrect, the logic for this was overly complex - it didn't need to be.

Closes #32.

### CODE REVIEW

File: src/paginate.ts [hunk 1]
```diff
@@ -90,17 +85,12 @@ export class Paginate<Model extends PrismaClientModel> {
         ? this.paginationArgs.pageIndex + 1
         : 1;
     const hasNextPage = page < totalPages;
-    const hasPrevPage =
-      count > 0 && page > 1
-        ? (page * this.paginationArgs.limit) / count - 1 === 0 ||
-          page - 1 === totalPages
-        : false;
```
Comment (sandrewTx08) [inline src/paginate.ts:97]: `hasPrevPage` intend to validate if the previous page has data, on this change its value will be `true` if `page` exceed max of pages.

Comment (jamesmoss) [reply src/paginate.ts:97]: I will update this logic to factor this in but it's worrying that my change didn't cause any of the other tests to fail

**Comment by jamesmoss** [comment]
Ive addressed your feedback, can this be merged?

**Comment by sandrewTx08** [comment]
Looks good.

**Pull request merged**